src/main/resources/assets/mwc/lang/en_US.lang

```diff
@@ -1119,10 +1116,9 @@ config.gameplay.statusBarPosition.property.name=Status Bar Position
 config.gameplay.statusBarPosition.property.tooltip=The screen position for the Status Bar
 
 config.category.entities.name=Entities Config
-
-config.category.rendering.name=Rendering
+config.category.rendering.name=Rendering Config
```

Comment:
No change it for all of it please, you open the config screen you know you are in the config screen you don't need "Config" to be plastered all over the screen

Reply:
Done!